quantum/rgblight.c

```diff
@@ -797,6 +805,15 @@ void rgblight_task(void) {
     // static light mode, do nothing here
     if ( 1 == 0 ) { //dummy
     }
+#ifdef RGBLIGHT_IDLE_ENABLE
```

Comment:
There's no need for multiple defines for this feature as you can just used `#if RGBLIGHT_IDLE_TIMEOUT > 0`

Reply:
Fair enough. Should I just use the rgb matrix variable or still use a separate one.

Reply:
I would keep the RGB Light define for this feature separate as there has been interest in running both RGB Light & Matrix at the same time.